content.en/websites/_index.md

```diff
@@ -42,6 +42,12 @@ for a direct comparison of their TTFB performance. This comparison reveals
 performance differences and provides insights into the efficiency of each
 protocol/implementation in different geographic areas.
 
+For Helia in the above graph, the different variations work out as follows:
+
+`Helia (TG)` - Helia in NodeJS: blocks fetched via [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/) only
+`Helia (DR)` - Helia in NodeJS: blocks fetched via [delegated routing](https://specs.ipfs.tech/routing/http-routing-v1/) with direct peer retrieval only 
+`Helia` - Helia in NodeJS: blocks fetched via [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/) & [delegated routing](https://specs.ipfs.tech/routing/http-routing-v1/) with direct peer retrieval
```

Comment:
```suggestion
`Helia` - Effectively the combination of "Helia (TG)" + "Helia (DR)".  Helia in NodeJS in parallel is:
 - delegating content, peer routing, and block entirely to [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/) AND
 - delegating content and peer routing to an external server using HTTP but then uses js-libp2p for direct peer retrieval. 
It's expected that this would yield more reliable results than "Helia (TG)" or "Helia (DR)" alone.

Important scenarios that aren't measured yet:
1. Helia in NodeJS where content, peer, and block retrieval are done entirely in a p2p fashion.  
2. The scenarios above but instead from within Chromium rather than NodeJS.  This will be coming soon per ISSUE X.
```